### PR TITLE
New typings for bunyan-format module

### PR DESCRIPTION
**Pull request by dex4er**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by RyanCavanaugh** [comment]
@dex4er Please fix the failures indicated in the Travis CI log.

File: types/bunyan-format/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// Type definitions for bunyan-format 0.2
+// Project: https://github.com/thlorenz/bunyan-format
+// Definitions by: Piotr Roszatycki <https://github.com/dex4er>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
```
Comment (andy-ms) [inline types/bunyan-format/index.d.ts]: Add a `// TypeScript Version: 2.1` comment on this line (just like `bunyan` has)

File: types/bunyan-format/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (andy-ms) [inline types/bunyan-format/tsconfig.json]: This should be `true`

File: types/bunyan-format/bunyan-format-tests.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import bunyan = require('bunyan');
```
Comment (andy-ms) [inline types/bunyan-format/bunyan-format-tests.ts]: This doesn't seem to show the two libraries interacting in any way?

Comment (dex4er) [reply types/bunyan-format/bunyan-format-tests.ts]: I simplified test for this module and now it doesn't use bunyan itself.

File: types/bunyan-format/tsconfig.json (hunk 2 above)
Comment (dex4er) [reply types/bunyan-format/tsconfig.json]: This is pasto because dts-gen doesn't generate correct tsconfig.json...

**Comment by andy-ms** [comment]
Thanks!

**Pull request merged**